apps/passport-server/src/services.ts

```diff
@@ -86,4 +86,5 @@ export async function stopServices(services: GlobalServices): Promise<void> {
   services.metricsService.stop();
   services.telegramService?.stop();
   services.persistentCacheService.stop();
+  await services.discordService?.stop();
```

Comment:
why await on this one?

Reply:
`discordService` contains field which is a promise that is necessary to resolve before `stop` can actually execute